pkgs/development/python-modules/django-crispy-forms/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv
+, fetchPypi
+, buildPythonPackage
+}:
+buildPythonPackage rec {
```

Comment:
Not sure whether you need to use `rec` here...

Reply:
I am pushing a test without rec. I had mixed results on some of my other recent PRs when it was removed.

Reply:
Follow up: removing `rec` causes bots to reject the PR because of bad output paths.